### PR TITLE
feat(Email Linking): Documentation

### PR DESCRIPTION
**Pull request by hrwX**

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Email Aggregation takes place in Contacts(by default), Customer and Supplier. All the Emails sent or received from a Contacts can be viewed in that Contact's Timeline. To enable Email Aggregation for a Customer or Supplier, do the following
+
+#### Step 1
+
+In a Contact, add Links for the Customer or Supplier respectively.
+
+<img class="screenshot" alt="Add Customer/Supplier in Contact" src="{{docs_base_url}}/assets/img/setup/email/contact-link.png">
+
+#### Step 2
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md]: H3s after H2s

File: erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+When an Email is sent to or received from the Customer or Supplier, the email is linked to the Customer or Supplier linked to the  Contact.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md]: good explanation

File: erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+<img class="screenshot" alt="With Filters" src="{{docs_base_url}}/assets/img/setup/email/email_aggregation.gif">
+
+### Automatic Email Linking to a Document
+
+Automatic Email Linking links an Email to the Document specified in the unique Email Address generated by the system for a Document. An Email sent or received usign the unique Email Address, the system will link that Email to the Document.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md]: typo: using

**Comment by hrwX** [comment]
@prasadarr Next time use `Suggestion` for typos and grammatical errors.

File: erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<!-- add-breadcrumbs -->
+# Linking Emails to Documents
+
+> Introduced in v12
+
+You can link an Email to any Document. You can link an Email to multiple Documents in ERPNext. This can be done in the following two ways.
```
[the remark this answers is not in the document]
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md]: read the sentence again, make the necessary changes

File: erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+- Email Aggregation in Contact, Customer and Supplier.
+- Automatic Email Linking.
+
+### Email Aggregation for Customer and Supplier
+
+Email Aggregation takes place in Contacts(by default), Customer and Supplier. All the Emails sent or received from a Contacts can be viewed in that Contact's Timeline. To enable Email Aggregation for a Customer or Supplier, do the following
```
[the remark this answers is not in the document]
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/setting-up/email/linking-emails-to-document.md]: first change was not made

**Pull request merged**